lib/decorators/placeholder.vue

```diff
@@ -236,6 +280,15 @@
       }
     },
     methods: {
+      toggleCollapse() {
+        if (this.isCollapsed) {
+          this.isCollapsed = false;
+          this.$el.parentNode.classList.remove('kiln-collapsed');
+        } else {
+          this.isCollapsed = true;
+          this.$el.parentNode.classList.add('kiln-collapsed');
+        }
+      },
```

Comment:
Would this work here?
```js
this.isCollapsed = !this.isCollapsed
// ...add/remove conditional here or classList.toggle()
```
More importantly, is there a more Vue-friendly way to do this than external DOM manipulation? Seems fragile

Reply:
Haha yeah I could just do that 🙃 (I forgot about `classList.toggle()`)

As for the fragility, I'll add a check to make it less brittle but there isn't really a more vue-y way to do this (since it's reaching out to a non-vue thing)

Reply:
it looks like you're passing the parentHeight to `$options`. Would it be useful to instead pass the whole parent reference? Esp if the height ever changes it would be better to track that within Vue anyway.

Reply:
The parent reference might change on re-render

Reply:
oh dang. and the Vue component can't be notified?